magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/LifecycleLimiter.kt

```diff
@@ -44,11 +44,15 @@ public class LifecycleLimiter : LifecycleOwner, LifecycleAware {
     attachToLifecycleWithMaxState(lifecycleAware, LifecycleLimit.NO_LIMIT, detachedState)
   }
 
-  public fun attachToLifecycleWithMaxState(lifecycleAware: LifecycleAware, maxState: LifecycleLimit, detachedState: LifecycleState = LifecycleState.Destroyed) {
+  internal fun attachToLifecycleWithMaxState(
```

Comment:
Curious of the thinking behind making this `internal`? Do we expect people consuming the library not to use this method? (We use it ourselves in the `DashboardTabsScreen` right now)